architecture/core_concepts/pods_and_services.adoc

```diff
@@ -38,6 +38,11 @@ maintain state when recreated. Therefore pods should usually be managed by
 higher-level link:deployments.html#replication-controllers[controllers],
 rather than directly by users.
 
+[IMPORTANT]
+====
+The maximum supported number of pods within an {product-title} cluster is 110.
```

Comment:
I don't know if we want to have a section on allowing over-rides here?  

Also might want to craft the language to be softer, as these limits can be stretched, but we want to give customers guidance, while allowing power-users to tinker with the engine. 

/cc @jeremyeder ?